### ISSUE TITLE
Psiminer doesn't work for large trees

### ISSUE THREAD

**Issue by psurkov** [JetBrains-Research/psiminer#24]
The java-med dataset has a file `train/stanfordnlp__CoreNLP/src/edu/stanford/nlp/process/PTBLexer.java` containing 76704 lines of code. IDEA parses this file as a single `PsiPlainText` element and psiminer does the same. I think the miner should skip such files with a warning

**Comment by SpirinEgor** [comment]
Agree, since this is not the miner responsibility it should skip them with proper logging. Do you plan to contribute to this feature?
